AcadeMeData/tests/test_msgboard_feature.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from AcadeMeData.models import User
+from AcadeMeData.forms import MessageForm
+
+
+@pytest.fixture
+def user_example():
```

Comment:
Any reason not to use `user_example` from `test_register` or from `test_user`? you should have all of those consolidated into a contest instead of rewriting them for each and every test module.

Reply:
What do you mean? importing them or merging the tests in to one file?
If importing is the answer then I have tried and failed so I changed it to this way

Reply:
Please refer this review also 
https://github.com/redhat-beyond/AcadeMe/pull/103#discussion_r775504679

If you do not understand, please try friends in slack. understand it and propose a code reply

Reply:
got it, fixed this in the recent push